LICENSE

```diff
@@ -1,7 +1,6 @@
-
 BSD 3-Clause License
 
-Copyright (c) 2022, QIIME 2 development team.
+Copyright (c) 2016-2022, QIIME 2 development team.
```

Comment:
This plugin didn't get started back in 2016, right? All of our plugins/packages have different copyright years - you should use the year the plugin was first created in.

Reply:
@thermokarst that's what I had originally, but q2lint failed because the LICENCE file didn't match the template - should the licence file itself contain only the years that the plugin has existed, or the years that the licence/QIIME 2 as a whole has existed?

Reply:
@thermokarst disregard, I'm not sure why this was a problem for me earlier - but it's resolved! Updating the copyright now.

Reply:
It should only be the years the plugin has existed.

> but q2lint failed because the LICENCE file didn't match the template

It said that because you had an extra line at the top of the LICENSE file. Once removed that error goes away.

The next error I see when running q2lint is that the wrong copyright year is listed in the `q2_fmt/_transformer.py` header.

I hope that helps.